### DISCUSSION TITLE
Wrap Text in Tooltips

### DISCUSSION THREAD

**Discussion by lgnashold** [palantir/blueprint#6541] [Q&A]
#### Feature request

Currently, if you have very long tooltip help text, it displays as a single large line. Instead, there should be a way for the text to wrap or a max-width to be specified.

**Comment by adidahiya** [comment]
@lgnashold use `<Tooltip popoverClassName="my-tooltip">` and style the popover content element: https://codesandbox.io/p/sandbox/suspicious-monad-rctwgj?file=%2Fsrc%2FCoreExample.tsx%3A12%2C48

![image](https://github.com/palantir/blueprint/assets/723999/92bf82c8-bdad-47e3-b1d8-a97e39db87ac)
